### ISSUE TITLE
Tooltips and what not

### ISSUE THREAD

**Issue by DankBudd** [NoFluxGiven/Dusk#4]
id like to note that most of, if not all of the skills with scepter upgrades dont tell you what the upgrade is until you buy the scepter

**Alexander** _(dragon knight)_
**| W** - has global cast range, doesnt state this anywhere

**Artificer** _(lone druid)_
**| R** - are the 0% bonuses really neccesary as tooltips?

**Fate** _(oracle)_
**| E** - has 2 damage tooltips rather than one for damage and one for heal, however, since both have the same values, they could be merged into one line as "damage/heal"

**Fury** _(ursa)_
**| R** - how much bonus damage is given based on missing health
**|**	- how much bonus damage is given if he dies during ult
**|**	- doesnt say what his base attack time is set to if he dies during ult

**Hawkeye** _(sniper)_
**| R** - Description says "travels for most of the map's length", yet its range (without scepter) is shorter than windrangers powershot
**|**	- my first time playing sniper i read that line and thought the spell was nearly global, and that the range value meant something else

**Horror** _(night stalker)_
**| W** - Description says "applies a blind and miss chance to nearby enemies"
**|**	as ive already told you, i didnt know that this took away the enemies vision when i played nightstalker. blind is generally associated with miss chance in dota, as squab said, tinkers laser applies a "blind" that causes the target to miss. perhaps reword it to something like "blinds and applies a miss chance to nearby enemies"
**|**	- tbh, i dont know if thats any better tho, it will probably still confuse some people.

**Iron Fist** _(dark seer)_
**| R** - needs duration tooltips on all his ults

**Lightning** _(phantom assassin)_
**| E** - doesnt state how much it slows by

**Lysander** _(kunkka)_
**| Q** - is refreshed when it crits, doesnt state this anywhere
**| W** - grants true sight, only says "gives vision"

**Mifune** _(juggernaut)_
**| R** - doesnt say what the radius around main target is

**Odin** _(abbadon)_
**| E** - does not clearly state who needs to attack who for the passive to take effect
**|**	- does not state duration of modifier

**Ouroboros** _(jakiro)_
**| E** - it has a cooldown tooltip, but theres nothing that would put it on cooldown

**Phantom** _(spectre)_
**| Q** - needs duration tooltip
**| W** - doesnt say how long it stuns for
**| E** - doesnt state cleave radius
**| R** - doesnt state crit chance

**Set** _(sandking)_	**_*Have not played this hero, these things might be clear ingame*_**
**| W** - has radius and distance tooltips?
**| R** - has scepter upgrade but doesnt say "Upgradable by Aghanim's Scepter"

**Shade** _(bounty hunter)_
**| Q** - needs radius and duration tooltips
**| E** - second half of the Description is a little messy
**| R** - needs radius tooltip

**The Lich** _(lich)_
**| W** - doesnt state slow amount or slow duration
**| E** - slow is a flat value in tooltips, should probably be %
**| R** - scepter slow is a flat value in tooltips, should probably be %

**Void Walker** _(rubick)_
**| E** - needs radius tooltip

**Vulcan** _(clockwork)_	_***Have not played this hero, these things might be clear ingame***_
**| W** - needs blink range tooltip (all blinks in dota have this)
**|**	- needs radius for nearby units
**| E** - would be nice if it told you the damage and slow amounts in the main heroes skill, but i guess only giving it to the unit works too.
**| D** - same as E, tell us what leveling the skill does
**| R** - this one is acceptable due to its already huge size, assuming that you tell us the dmg and what not in the units skills

**War** _(axe)_ **_*Have not played this hero, these things might be clear ingame*_**
**| E** - attack speed supposed to be %?

And FINALLY
--
exalted scythe of vyse doesnt state stat bonuses

**Comment by darklordabc** [comment]
What a trash game, the developer really has to cancel this project.

**Comment by NoFluxGiven** [comment]
thanks for this report dankbudd, will get on those
